modules/ROOT/pages/index.adoc

```diff
@@ -23,6 +23,162 @@ Anypoint Platform for PCF uses the following services:
+****** Specify the version of the Mule runtime per application.
+****** Specify the amount of memory for each application replica.
+****** Cluster Mule runtimes by setting up an externalized HazelCast service broker.
+****** Configuring the application endpoint name.
+****** Start, stop, and delete Mule applications.
+****** View monitoring data and set alerting via PCF metrics.
+****** Re-deploy without downtime, with support for blue/green deployments.
+****** Automatic failover of Mule applications.
+
+===== API Policy Management 
+API policies are supported with the following additional setup:
+
+For PCE versions 1.6.x and 1.7.x:
+
+. Deploy an app:
+
+.. Deploy an app using `cf push <app>` with the manifest file or typical `cf` commands.
+.. After the app is created, run `cf app <app>` and make note of the app URL.
+. Deploy a proxy application:
+               
+.. Use the PCE instance to get properties.
++
+----
+-Dapi.name=<customerApp name>     
+-Dapi.version=<version of the app ex: 1.0>    
+-Dimplementation.host=<url of customerapp ex: springboot.apps.santafesprings.cf-app.com>
+-Dimplementation.port=80 
+-Dimplementation.path=/ 
+-Danypoint.platform.client_id=<clientId property that you have for the org ex: cdbff39bfa4741fb9bb531ba6102a578> 
+-Danypoint.platform.client_secret=<ClientId secret of the org ex: 56bB958Bc257464B9fF0A9F9E6Af9890> 
+-Danypoint.platform.platform_base_uri=< ex: https://qax.anypoint.mulesoft.com/apiplatform>
+-Danypoint.platform.coreservice_base_uri=< ex: https://qax.anypoint.mulesoft.com/accounts >
+-Danypoint.platform.on_prem=true 
+-Danypoint.platform.contracts_base_uri=< ex: https://qax.anypoint.mulesoft.com/apigateway/ccs>
+----
+
+.. After gathering all the properties, deploy the proxy:
++
+----
+. cf push <proxy> -b <mulebuildpackname> -p <proxy app that mulesoft provided> --no-start
+. cf set-env <proxy-app> mcm_anypoint_metering_plan_unlimited true
+. cf set-env <proxy-app> MULE_JAVA_OPTS "-Dapi.name=muleapp -Dapi.version=1.0
+-Dimplementation.host=muleapp.apps.santafesprings.cf-app.com -Dimplementation.port=80
+-Dimplementation.path=/ -Danypoint.platform.client_id=0f2ea15d88fa452e8ca1a5f4bb724728
+-Danypoint.platform.client_secret=BbCA70Bb056F437e94C707B67aF4284D
+-Danypoint.platform.platform_base_uri=https://161-171-upgrade-2056596225.us-east-2.elb.amazonaws.com/apiplatform
+-Danypoint.platform.coreservice_base_uri=https://161-171-upgrade-2056596225.us-east-2.elb.amazonaws.com/accounts -Danypoint.platform.on_prem=true
+-Danypoint.platform.contracts_base_uri=https://161-171-upgrade-2056596225.us-east-2.elb.amazonaws.com/apigateway/ccs
+
+. cf start <proxy>
+----
+
+.. Tail the logs by running `cf logs <proxy>`.
+
+***  The 2018-11-13T10:07:09.52-0800 [APP/PROC/WEB/0] OUT INFO  2018-11-13 18:07:09,494 [agw-policy-polling.01] com.mulesoft.module.policies.lifecyle.GateKeeper log should indicate that API <proxyappname> <version> is now unblocked.
```

Comment:
extra space; also, please reread this item to verify that it is understandable and clear to your intended audience.
```suggestion
*** The 2018-11-13T10:07:09.52-0800 [APP/PROC/WEB/0] OUT INFO  2018-11-13 18:07:09,494 [agw-policy-polling.01] com.mulesoft.module.policies.lifecyle.GateKeeper log should indicate that API <proxyappname> <version> is now unblocked.
```